docker-test.sh

```diff
@@ -59,30 +18,40 @@ else
     export FORCE_RECREATE_ARGUMENT="--force-recreate"
 fi
 
+function start_node() {
+    PUBLIC_API_EXTERNAL_PORT=$1 PRIVATE_NETWORK=$4 NODE_NAME=$2 NODE_IP=$3 docker-compose -p orbs-$2 -f $VOLUMES -f docker-compose.test.yml up -d $FORCE_RECREATE_ARGUMENT
```

Comment:
Could you make sure that failed starts are properly handled?

Reply:
@netoneko can you help me achieve that?